lie/lie/options.py

```diff
@@ -42,12 +42,14 @@ def get_eps(self, ltype: str, attr: str, dtype: torch.dtype) -> float:
     def reset(self) -> None:
         self.so3_near_pi_eps_float32 = 1e-2
         self.so3_near_zero_eps_float32 = 1e-2
+        self.so3_d_near_zero_eps_float32 = 2e-1
```

Comment:
Flagging that these new ones would need to be added to options on the theseus side. Unless we refactor to something like the wrapper version based on [this comment thread](https://github.com/facebookresearch/theseus/pull/528#discussion_r1239127697).